src/Explorer/Tables/TableDataClient.ts

```diff
@@ -523,8 +523,10 @@ export class CassandraAPIDataClient extends TableDataClient {
   }
 
   private setAuthorizationHeader: (xhr: XMLHttpRequest) => boolean = (xhr: XMLHttpRequest): boolean => {
-    const authorizationHeaderMetadata: ViewModels.AuthorizationTokenHeaderMetadata = getAuthorizationHeader();
-    xhr.setRequestHeader(authorizationHeaderMetadata.header, authorizationHeaderMetadata.token);
+    const authorizationHeaderMetadata = getAuthorizationHeaders();
+    Object.entries(authorizationHeaderMetadata).forEach((value, index, array) => {
```

Comment:
It looks like we are not using `index` and `array` so we should just remove them from the parameters.